Index: app/src/main/java/com/example/chattingapp/database/UsersDatabase.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/chattingapp/database/UsersDatabase.java	(date 1464074081000)
+++ app/src/main/java/com/example/chattingapp/database/UsersDatabase.java	(revision )
@@ -40,7 +40,6 @@
                 + FIRSTNAME + " TEXT, " + LASTNAME + " TEXT, " + PROFILEPIC + " TEXT " + ")";
 
         db.execSQL(CREATE_TABLE);
-        Log.i("onCreate","nn");
     }
 
     public void addUser(UserModel user) {
